src/cc_catalog_airflow/dags/provider_api_scripts/walters_art_museum.py

```diff
@@ -0,0 +1,209 @@
+"""
+Content Provider:       Walters Art Museum
+
+ETL Process:            Use the API to identify all CC licensed images.
+
+Output:                 TSV file containing the images and the
+                        respective meta-data.
+
+Notes:                  http://api.thewalters.org/
+                        Rate limit: 250000 Per Day Per Key
+"""
+
+import os
+import logging
+import common.requester as requester
+import common.storage.image as image
+from util.loader import provider_details as prov
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.DEBUG

```

Comment:
Please turn down the logging level.

```suggestion
    level=logging.INFO

```